app/models/relationship.rb

```diff
@@ -380,6 +398,10 @@ def move_explainers_to_source
     end
   end
 
+  def apply_status_to_children
+    Relationship.delay_for(1.second, { queue: 'smooch_priority' }).replicate_status_to_children(self.source_id, User.current&.id, Team.current&.id)
+  end
```

Comment:
Sawy, can you please explain why we need to replicate the status for all children when a single relationship is saved? Is it only because it's "easier" (and we ignore if the statuses are the same anyway)?

Reply:
Yes Caio as I am using same method but you are right I should update the target item status only